server/lib/controllers/articleController.js

```diff
@@ -31,7 +35,30 @@ articleController.getOne = async (req, res) => {
     const article = await _Article.default.findOne({
       _id: req.params.id
     });
-    res.send(article);
+
+    try {
+      var likes = await _Like.default.find({
```

Comment:
why are you using var instead of const?

Reply:
There is no specific reason. I thought that I can use any of them as long as it can fit in. Should I use Const?

Reply:
remember we are using es6